scripts/test

```diff
@@ -13,19 +19,23 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
+        if [ "${1:-}" = "--jenkins" ]; then
+            ANGULAR_LINT_CMD="lint:ci"
+        else
+            ANGULAR_LINT_CMD="lint"
+        fi
+
```

Comment:
Since `scripts/update` doesn't build a tagged image, I think it'd be useful to build one here.

```bash
GIT_COMMIT=${GIT_COMMIT} docker-compose \
       -f docker-compose.yml \
       -f docker-compose.test.yml build
```